PROJECT_LIFECYCLE.md

```diff
@@ -96,6 +96,13 @@ moving to a different stage.
 Due to the nature of sandbox projects (eg. experimentation), project can be moved to archive stage at any point in
 time if the maintainers deem it necessary.
 
+You should use the ![Sandbox badge](https://img.shields.io/badge/Stage-Sandbox-yellow.svg) badge to show the project stage.
```

Comment:
Where should the badge be placed? Over or under the project logo? And also on the community PROJECTS.md page I guess?, or should there even by a link from PROJECTS.md to a file in each project stating the maturity level, so that the same link could be used in the projects README.md file. That way we would not risk having different maturity levels stated in a projects README.md and in the community PROJECTS.md document

Reply:
Added a description in 08037d6. What I recall from the TOC discussion we didn't want to put the logo on the PROJECTWS.md page but my memory could fail me

Reply:
I don't know if it was decided or not, but I would actually prefer the badge as opposed to a text string in the PROJECTS.md. The badge gives a stronger visual cue and might confers the meaning/purpose better.